### PR TITLE
coq: relax lablgtk3-sourceview3 dependency

### PR DESCRIPTION
**Pull request by RalfJung**
Cc https://github.com/ocaml/opam-repository/pull/16876#discussion_r460521690

### CODE REVIEW

File: packages/coqide/coqide.8.12.0/opam [hunk 1]
```diff
@@ -14,7 +14,7 @@ using the Coq proof assistant.
 
 depends: [
   "coq" {= version}
-  "lablgtk3-sourceview3" {>= "3.1.0"}
+  "lablgtk3-sourceview3" {>= "3.0"}
```
Comment (Blaisorblade) [inline packages/coqide/coqide.8.12.0/opam:17]: I'd have thought this bound was trivially satisfied?

Comment (Blaisorblade) [reply packages/coqide/coqide.8.12.0/opam:17]: See e.g. https://github.com/ocaml/opam-repository/tree/master/packages/lablgtk3-sourceview3

Comment (RalfJung) [reply packages/coqide/coqide.8.12.0/opam]: Coming from Rust, an unconstrained dependency is a complete no-go... but sure.

Comment (Blaisorblade) [reply packages/coqide/coqide.8.12.0/opam]: Thanks. And sorry that we looped back here... (My initial question was still a valid question, but I could have spent more looking up the answer 😅).

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**